new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java

```diff
@@ -64,6 +64,29 @@ public static boolean doesPodExist(String namespace, String domainUid, String po
     return podExist;
   }
 
+  /**
+   * Checks if a pod does not exist in a given namespace in any state.
+   * @param namespace in which to check for the pod existence
+   * @param domainUid the label the pod is decorated with
+   * @param podName name of the pod to check for
+   * @return true if pod does not exists otherwise false
+   * @throws ApiException when there is error in querying the cluster
+   */
+  public static boolean doesPodDeleted(String namespace, String domainUid, String podName) throws ApiException {
```

Comment:
wasPodDeleted is a better name, but this method does not actually test that a pod was deleted, it just just checks that the pod does not exist

Reply:
How about changing the method name to podDoesNotExits ( in contrast to podExists)